feat(em): update to monthly billing (#4603)
scaleway/docs-content: pages/elastic-metal/how-to/change-billing-period.mdx

---
meta:

>   title: How to change the billing period for Elastic Metal Servers

Reviewer: ```suggestion
  title: How to change the billing period for Elastic Metal servers
```

>   title: How to change the billing period for Elastic Metal servers

Author: ```suggestion
  title: How to change from hourly to monthly billing
```

  description: This page explains how to change the billing period for Scaleway's Elastic Metal servers.
content:

>   h1: How to change the billing period for Elastic Metal Servers

Reviewer: ```suggestion
  h1: How to change the billing period for Elastic Metal servers
```

>   h1: How to change the billing period for Elastic Metal servers

Author: ```suggestion
  h1: How to change from hourly to monthly billing
```

By switching to monthly billing, you can benefit from significant discounts compared to the hourly pricing model.
This provides you with cost savings for longer-term server rentals and allows you to retain your existing server configuration without requiring a new setup.

<Macro id="requirements" />
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- An [Elastic Metal server](/bare-metal/how-to/create-server/)

1. Click [Elastic Metal](https://console.scaleway.com/elastic-metal/servers) in the **Bare Metal** section of the [Scaleway console](https://console.scaleway.com) side menu. The Elastic Metal dashboard displays.
2. Click the server you want to configure. The server's information page displays.
3. Scroll down to the **Billing preferences** section. Here, you can see information about your current billing option. Click **Update to monthly billing**. A pop-up displays.

> 4. Review the server and optipons details and their new monthly pricing. Note that switching to monthly billing incurs a one-time commitment fee, which is charged immediately.

Reviewer: ```suggestion
4. Review the server and options details and their new monthly pricing. Note that switching to monthly billing incurs a one-time commitment fee, which is charged immediately.
```

    <Message type="important">

>         - Once switched to monthly billing, reverting to hourly billing is not possible.

Reviewer: ```suggestion
        - Once switched to monthly billing, it is not possible to revert to hourly billing.
```

>         - Once switched to monthly billing, it is not possible to revert to hourly billing.

Reviewer 2: Maybe this document should therefore be called "How to change from hourly to monthly billing"? Since it can't be done in the other direction?

Or else you could say explicitly in the introduction that it is possible to change from hourly to monthly but not vice versa.
